recipes/openmm-ml/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "openmm-ml" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-ml/archive/28b4fadd761221fbe085570faac3e95a2d0be4e0.tar.gz
+  sha256: 913c48641c5fdcbdc6d7ccf14cd33a20ad432c781455e331da86a7a8cc455330
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - openmm >=7.5
```

Comment:
This won't work with OpenMM 7.5.  In fact, it won't even really work with 7.7.  To get things fully functional, we need [these fixes](https://github.com/openmm/openmm/milestone/6?closed=1) that will be included in the next release.

Reply:
So I know it isn't out yet, but we will need `openmm >=7.7.1` then?